user/enterprise/troubleshooting-guide.md

```diff
@@ -0,0 +1,162 @@
+---
```

Comment:
Did you refactor this beyond the rename and the contact form? I think not right?

Reply:
I did add the redirect so that links pointing to the old `operations-manual` would still work. I only added it based on the assumption that we would want to preserve old links but please let me know if that's generally not a concern here.